src/translation/english.c

```diff
@@ -150,14 +150,14 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_SHOW_ADVISOR_CHIEF, "Chief advisor"},
     {TR_HOTKEY_SHOW_ADVISOR_HOUSING, "Housing advisor"},
     {TR_HOTKEY_TOGGLE_OVERLAY, "Toggle current overlay"},
-    {TR_HOTKEY_SHOW_OVERLAY_RELATIVE, "Show relative overlay"},
-    {TR_HOTKEY_SHOW_OVERLAY_WATER, "Show water overlay"},
-    {TR_HOTKEY_SHOW_OVERLAY_FIRE, "Show fire overlay"},
-    {TR_HOTKEY_SHOW_OVERLAY_DAMAGE, "Damage overlay"},
-    {TR_HOTKEY_SHOW_OVERLAY_CRIME, "Crime overlay"},
+    {TR_HOTKEY_SHOW_OVERLAY_RELATIVE, "show Relative overlay"},
```

Comment:
I think it'd be better if you kept the first word capitalized, so it'd be uniform across all the options.

Reply:
The reason I changed it was because when capitalizing every occurrence of "Show" it's that word that gets emphasized in the long list of overlay options. By capitalizing the actual overlay name, e.g.  "show Health overlay", in the config window it actually looks like a list of overlays, rather than a list of the word "Show". Or, alternatively, maybe remove the word "Show" altogether as is the case currently in 3.1 for the Damage, Crime and Problems overlay?
Let me know which is your preferred option and I'll amend accordingly.

Reply:
Removing the word 'show' sounds good, I feel that paradoxically leaving the first word uncapitalized otherwise draws attention to it.

Reply:
The overlay hotkey labels in 3.1 aren't consistent in that we have "Show relative overlay", "Show water overlay" and "Show fire overlay", but also just "Damage overlay", "Crime overlay", "Problems overlay". I notice that the overlay labels for the advisors don't have the word "Show" at all.

Is it ok if I make the overlay labels consistent with the advisors labels and just have the following to make it all consistent?...

Water overlay
Fire overlay
Crime overlay
etc ?

If you let me know, I'll get the changes in today.

Reply:
Done.